.ci/build_packages/tests.sh

```diff
@@ -148,8 +148,15 @@ EOF
         IDLE_TIME=$((IDLE_TIME+1))
     done
     pytest -v /paho-mqtt-testing/interoperability/test_client/V5/test_connect.py::test_basic
+    export DEBUG=1
     # shellcheck disable=SC2009 # pgrep does not support Extended Regular Expressions
-    emqx stop || kill "$(ps -ef | grep -E '\-progname\s.+emqx\s' |awk '{print $2}')"
+    ps -ef | grep -E '\-progname\s.+emqx\s'
+    if ! emqx 'stop'; then
+        echo "ERROR: failed_to_stop_emqx_with_the_stop_command"
+        cat /var/log/emqx/erlang.log.1 || true
+        cat /var/log/emqx/emqx.log.1 || true
```

Comment:
maybe cat more logs not just `emqx.log.1`?

Reply:
it's only start then stop, no test steps in between